apps/prairielearn/elements/pl-details/pl-details.css

```
@@ -0,0 +1,38 @@
+  transform: unset;
+}
+
+details.accordion-item[open] .accordion-button::after {
+  background-image: var(--bs-accordion-btn-icon);
+  transform: var(--bs-accordion-btn-icon-transform);
+}
+
+/* Taken from https://nerdy.dev/open-and-close-transitions-for-the-details-element */
+@media (prefers-reduced-motion: no-preference) {
+  details.accordion-item {
+    interpolate-size: allow-keywords;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)